.github/workflows/ci.yaml

```
@@ -20,7 +21,7 @@ jobs:
   setup:
     name: Setup
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: ${{ env.DEFAULT_TIMEOUT }}
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)